pennylane/operation.py

```diff
@@ -1023,7 +1026,14 @@ def __init__(self, *params, wires=None, do_queue=True, id=None):
 
         self.data = [np.array(p) if isinstance(p, (list, tuple)) else p for p in params]
 
-        if do_queue:
+        if do_queue is not None:
+            do_queue_deprecation_warning = (
+                "The do_queue keyword argument is deprecated. "
+                "Use qml.queuing.QueuingManager.stop_recording()"
```

Comment:
```suggestion
                "The do_queue keyword argument is deprecated. Instead of setting "
                "it to false, use qml.queuing.QueuingManager.stop_recording()"
```